earth_enterprise/src/third_party/numpy/SConscript

```diff
@@ -52,7 +52,7 @@ numpy_extract = numpy_env.Command(
                        numpy_target))])
 
 # [2] Build numpy.
-python_cmd = '%s/bin/python' % root_dir
+python_cmd = numpy_env['python_bin']
```

Comment:
so this picks up the python bin path from numpy?  I assume numpy detects and sets it from somewhere...

Reply:
This is a variable copied from the parent scons tasks. The numpy_env in this case is a variable containing the build environment.